src/redux/configureStore.js

```diff
@@ -8,8 +9,9 @@ const rootReducer = combineReducers({
 });
 
 /* eslint-disable */
-const store = createStore(rootReducer,
-window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
-);
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
- [ ] There is no need to apply an extra store enhancer as you are using the default enhancer the `applymiddleware`, Please remove the `composeEnhancers`. It's also throwing an error on the console and the app is not running.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/169643756-63a121d6-41d4-480d-a082-2257a88c486e.png)

Reply:
Hi @thecodechaser, I have removed the redux dev tool config. Everything is working fine